flamby/datasets/fed_camelyon16/dataset_creation_scripts/tiling_slides.py

```diff
@@ -287,8 +359,15 @@ def main(batch_size, num_workers_torch, tile_from_scratch, remove_big_tiff, outp
     parser.add_argument(
         "--remove-big-tiff",
         action="store_true",
-        help="Whether or not to remove the original slides images that take \
-            up to 800G, after computing the features using them.",
+        help=(
+            "Whether or not to remove the original slides images that take "
+            "up to 800G, after computing the features using them."
+        ),
+    )
+    parser.add_argument(
+        "--use-phikon",
```

Comment:
I would call it --use-ssl-features instead of --use-phikon which might not be obvious to a newcomer.